backend/api/Areas/Tools/Controllers/GeoServerController.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Pims.Core.Http;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Pims.Api.Areas.Tools.Controllers
+{
+    /// <summary>
+    /// GeoServerController class, provides proxy endpoints to integrate with the PSP GeoServer.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [Area("tools")]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/[area]/geoserver")]
+    [Route("[area]/geoserver")]
+    public class GeoServerController : ControllerBase
+    {
+        #region Variables
+        private IConfiguration _configuration;
+        private readonly IProxyRequestClient _proxyRequestClient;
+        #endregion
+
+        public GeoServerController(IConfiguration configuration, IProxyRequestClient proxyRequestClient)
+        {
+            _configuration = configuration;
+            _proxyRequestClient = proxyRequestClient;
+        }
+
+
+        /// <summary>
+        /// Proxy a request to the PSP GeoServer.
+        /// </summary>
+        /// <returns>the GeoServer response.</returns>
+        [HttpGet("{**rest}")]
+        [SwaggerOperation(Tags = new[] { "{**rest}" })]
+        public async Task<HttpResponseMessage> Proxy(string rest)
+        {
+            var queryString = this.Request.QueryString.Value;
+            var geoServerProxyUrl = _configuration.GetValue<string>("GeoServerProxyUrl");
+            var proxiedUrl = $"{geoServerProxyUrl}{rest}{queryString}";
+
+            return await _proxyRequestClient.ProxySendAsync(Request, proxiedUrl);
```

Comment:
This doesn't work currently, even with the Authorization header in place, the request just 302s to https://logontest7.gov.bc.ca/

Reply:
However, I have tested that the auth header is present in the forwarded request.